bin/apisix

```diff
@@ -998,19 +998,19 @@ local function init_etcd(show_output)
         local errmsg = string.format("got malformed version message: \"%s\" from etcd", res)
         local body, _, err = dkjson.decode(res)
         if err then
-            io.stderr:write(errmsg)
+            error(errmsg)
             return
         end
 
         local cluster_version = body["etcdcluster"]
         if not cluster_version then
-            io.stderr:write(errmsg)
+            error(errmsg)
             return
         end
 
         if compare_semantic_version(cluster_version, min_etcd_version) then
-            io.stderr:write("etcd cluster version ", cluster_version,
```

Comment:
I think it's strange here to use `error` since it exposes the coroutine stack traceback besides the error message, which is useless for administrator. Since it's not a programming fault or some systemtical errors, i think output error message to stderr is better.

Reply:
thank you for the suggestion, but simply output is error is very likely to be overlooked, like in #2342. 
Replace `error` with `write`+`exit`, to not expose the traceback